### DISCUSSION TITLE
SolverError when starting solution

### DISCUSSION THREAD

**Discussion by Johannes-hub22** [pybamm-team/PyBaMM#2327] [Q&A]
Hi guys, 

I am trying to simulate battery degradation using the O'Kane dataset. My initial idea is to replicate Figure 1 of the recent O'Kane et al publication (2022).  I often get a SolveError when simulating an experimental cycle using initial (starting) solution from the prev. solutions. This is not dependent on the parameter value (e.g. Dsol for SEI) as I have run into this on numerous simulations / investigations.

Here is the error:
SolverError: Could not find acceptable solution: solver terminated successfully, but maximum solution error (0.19144467801435194) above tolerance (1e-06)

I do not want to disable the eSOH functionality as I would like to investigate SOH etc.

Is there a workaround for this?

**Comment by valentinsulzer** [comment]
Please post the full code that causes the error

**Reply by Johannes-hub22** [reply]
import pybamm
import matplotlib.pyplot as plt
import numpy as np
import os
import scipy

pybamm.set_logging_level("NOTICE")

exp_cycles = 2

save_exp = 1


option={"SEI":"solvent-diffusion limited"}

model = pybamm.lithium_ion.DFN(options=option)


param = pybamm.ParameterValues("OKane2022")



param.update({"Outer SEI solvent diffusivity [m2.s-1]": 7.5E-21})


exp_ini = pybamm.Experiment(
[(
"Hold at 4.2V until 50mA ",
"Rest for 4 hours "
)],
period="30 seconds" 
)


exp_char = pybamm.Experiment(
[
    (
        "Discharge at 0.1C until 2.5 V", 
        "Charge at 0.3C until 4.2 V", 
        "Hold at 4.2 V until 50 mA",
    )
], 
period="30 seconds" 
)


exp_cycling = pybamm.Experiment(
[
    (
        "Discharge at 1C until 2.5 V", 
        "Hold at 2.5 V until 50mA",
        "Rest for 300 seconds",
        "Charge at 1C until 4.2 V", 
        "Hold at 4.2 V until 50 mA",
        "Rest for 300 seconds"
    )
] * exp_cycles, 
termination="20% capacity" # SOH_C of the battery
)

var_pts = {
    "x_n": 50,  # negative electrode
    "x_s": 40,  # separator 
    "x_p": 30,  # positive electrode (not interested in this for SEI growth)
    "r_n": 120, # negative particle
    "r_p": 40,  # positive particle (not interested in this for SEI growth)
}

exp = exp_ini
sim_ini = pybamm.Simulation(model, experiment=exp, parameter_values=param, var_pts= var_pts, )
sol_ini = sim_ini.solve()

exp = exp_char
sim = pybamm.Simulation(model=model, experiment=exp, parameter_values=param, var_pts= var_pts)
sol_k = sim.solve(starting_solution=sol_ini, save_at_cycles=save_exp)

exp = exp_cycling
sim = pybamm.Simulation(model=model, experiment=exp, parameter_values=param, var_pts= var_pts)
sol = sim.solve(starting_solution=sol_k, save_at_cycles=save_exp)

**Reply by valentinsulzer** [reply]
btw you can use three backticks to format your code, starting with ```python3 and ending with ```
```python3
import pybamm
model = ...
```

**Reply by Johannes-hub22** [reply]
Thanks. I am still a beginner and operating on Spyder IDE :)

**Reply by Johannes-hub22** [reply]
@tinosulzer: One more observation... It seems like the simulation is sensitive when updating parameters.

When I update the "Dead lithium decay constant [s-1]" to the default value of 4E-07, it throws out the same error.
i.e.
param.update({"Dead lithium decay constant [s-1]": 4E-07})

**Comment by valentinsulzer** [comment]
Should be fixed by #2355
